CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+Changelog
+=========
+Version 0.1.0
+
+				Initial release
```

Comment:
In this case, it actually looks like using an unordered list would be more apt, given this is a list of changes.
The version headings can also be converted to markdown headings with a `##` prefix

```markdown
## Version 0.1.0
- Add Submission Received Message to xblock
- Complete pylint and pep8 compliance
- Change minimum word count from 0 to 1
- Process score only if word count is valid
```

would render

## Version 0.1.0
- Add Submission Received Message to xblock
- Complete pylint and pep8 compliance
- Change minimum word count from 0 to 1
- Process score only if word count is valid